src/components/formControlSection/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC, PropsWithChildren } from 'react';
+import {
+  Box,
+  FormControl as ChakraFormControl,
+  Flex,
+  FormErrorMessage,
+} from '@chakra-ui/react';
+
+import Tooltip from '../tooltip';
+import Text from '../text';
+
+import { InformationIcon } from '../icons';
+
+export type Props = {
+  id: string;
+  errorMessage?: string;
+  label?: string;
+  hint?: string;
+  tooltip?: string;
+};
+
+const FormControlSection: FC<PropsWithChildren<Props>> = ({
```

Comment:
You will need to add this new component in the chakra migration PR stack. We also need to create the two tech20 tickets for these two components for the migration